src/services/ui/src/pages/dashboard/Lists/spas/consts.tsx

```diff
@@ -41,6 +41,7 @@ export const TABLE_COLUMNS = (props?: {
   {
     field: "actionType.keyword",
     label: "Action Type",
+    visible: false,
```

Comment:
Do we want to keep the row but hide it, or legitimately remove the rows. I don't believe action type or source are available columns in current OneMAC spec.